odd_collector/adapters/snowflake/client.py

```diff
@@ -260,6 +273,17 @@ def get_raw_stages(self) -> List[RawStage]:
         with self.connect() as cursor:
             return self._fetch_something(RAW_STAGES_QUERY, cursor, RawStage)
 
+    def _get_clustering_keys(self, tables: List[Table]) -> Dict:
+        res = {}
+        regex = r"\(([^()]*)\)"
```

Comment:
Could you please write comment what are groups did you parse? or use named groups